master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Configuring IP-in-IP
+---
+
+If your network fabric performs source/destination address checks 
+and drops traffic when those addresses are not recognized, it may be necessary to
+use IP-in-IP encapsulation of the inter-workload traffic.
+  
+This is often the case for public-cloud environments where you have limited control
+over the fabric and you are not peering your Calico BGP network with the rest of the
```

Comment:
If you have IP hops and your Calico network is peered with your fabric then the workload IPs will be advertised to the rest of your network and your routers will know how to get to those IPs.